src/main/java/persistence/entity/PersistenceContextImpl.java

```diff
@@ -26,6 +26,15 @@ public void removeEntity(Object entityObject) {
         entityCache.remove(cacheKey);
     }
 
+    @Override
+    public boolean isEntityExists(Object entityObject) {
+        Class<?> entityClass = entityObject.getClass();
+        Object entityId = EntityPrimaryKey.build(entityObject).keyValue();
+        Object existingEntity = getEntity(entityClass, entityId);
+
+        return existingEntity != null;
+    }
```

Comment:
`getEntity` 의 결과가 null 인 것 보단, 이렇게 확인 할 수 있지 않을까요? :)
```suggestion
    @Override
    public boolean isEntityExists(Object entityObject) {
        Class<?> entityClass = entityObject.getClass();
        Object entityId = EntityPrimaryKey.build(entityObject).keyValue();
        final EntityKey entityKey = createEntityKey(entityClass, entityId);

        return entityCache.containsKey(entityKey);
    }
```